enterprise/models.py

```diff
@@ -942,7 +954,12 @@ def contains_content_items(self, content_id_field_name, content_id_values):
             updated_content_filter[content_id_field_name] = content_id_values
             results = CourseCatalogApiServiceClient().get_search_results(updated_content_filter)
             if results:
-                content_ids_in_catalog = {content[content_id_field_name] for content in results}
+                for content in results:
+                    if content.get('content_type', '') == 'courserun' and content.get('has_enrollable_seats', False):
```

Comment:
No need to use `get` here, course run items will always have the `content_type` and `has_enrollable_seats` keys. Same thing below with programs.